tests/test_hf_api.py

```diff
@@ -472,6 +471,20 @@ def test_list_models(self):
         self.assertGreater(len(models), 100)
         self.assertIsInstance(models[0], ModelInfo)
 
+    @with_production_testing
+    def test_list_models_author(self):
+        _api = HfApi()
+        models = _api.list_models(author="google")
```

Comment:
These tests aren't actually ensuring that the query is returning items that have `author="google"`, just that something was returned. I'd recommend changing each of these to look something like the following:

```python
# After your previous two
self.assertTrue("google" in models[0].author)
```
(And then same for where you would expect bert to be)

Reply:
done!